app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::ItemsController < Api::ApiController
+  respond_to :json
+
+  before_action :authenticate
+
+  def create
+
+    @list = List.find(params[:list_id])
+    @items = @list.items
+    @item = @list.items.build(item_params)
+    @item.list = @list
+
+    if @item.save
+      render json: @item
+    end
+  end
+
+  def destroy
+    @list = List.find(params[:list_id])
```

Comment:
Scope `find` query off of `@user.lists` rather than `List`. Currently a logged-in attacker could, if they brute force guess the right ids, destroy items from a victim's list.